src-tauri/src/main.rs

```diff
@@ -145,7 +147,7 @@ fn main() {
             apply_launcher_pr,
             apply_mods_pr,
             get_launcher_download_link,
-            close_application,
+            close_application
```

Comment:
You can keep the trailing comma ;)

```suggestion
            close_application,
```

Reply:
I might have removed it by mistake while trying to solve the clippy errors.
I just couldn't import both methods like others...

Reply:
Yeah, I noticed that while trying as well. Super weird :/

Reply:
> I just couldn't import both methods like others...

For some reason importing it this way worked for me

https://github.com/R2NorthstarTools/FlightCore/pull/314/files#diff-2f5e0a90d4195e9986f5e24928dce16b59a80a2cf30f7059b38d55bd7d1eff69R156

Reply:
I think the `use app::*;` is what's causing the issue. I'll have to look into getting rid of that again

https://github.com/R2NorthstarTools/FlightCore/pull/301/files#diff-2f5e0a90d4195e9986f5e24928dce16b59a80a2cf30f7059b38d55bd7d1eff69R14